Create install_fenics-hpc_0.8.4-intel-2016-impi-5.1.3.223.sh

victorsndvg opened the pull request:
Install fenics-hpc with intel compilers

victorsndvg committed:
Create install_fenics-hpc_0.8.4-intel-2016-impi-5.1.3.223.sh

Install fenics-hpc with intel compilers

emepetres reviewed (changes requested):
Please Victor could you also update the README to explain a little bit about this file and how it should be used (e.g. instead of the actual install file, is a complement, an improvement over the previous one?)

victorsndvg commented:
Ok @emepetres ,

Is a new installation of FEniCS-HPC with intel compilers and using another installation of PETSc library in order to get better performance.

Ok, I agree with you :+1:   I will create a folder with a README.md file and some scripts to run the cube demo with this new installation ASAP.

victorsndvg committed:
Organize scripts and update README.md

victorsndvg commented:
Hi @emepetres ,

I've added some scripts and change the *README.md* scripts.

The wiki document was also updated.

Let me know if this changes are OK for you

ncde committed:
dolfin parameters fix, gts added to gnu setting
